composer.json

```diff
@@ -29,7 +29,8 @@
     "require-dev": {
         "doctrine/lexer": "^1.0.1",
         "doctrine/orm": "^2.6",
-        "phpunit/phpunit": "^7.0|^8.0",
+        "egulias/email-validator": "^2.1.10",
```

Comment:
Where's this being used? 🤔

Reply:
It is used by `swiftmailer/swiftmailer` and I had a bug when testing with ``--prefer-lowest`.
By the way, why `swiftmailer/swiftmailer` in the dev reqs? Is it possible to review the associated tests and mock what should be mocked?

Reply:
It's used here https://github.com/scheb/2fa/blob/master/src/email/Mailer/AuthCodeMailer.php#L26 and in the related test case.

But since we don't want to force people into using that specific mailer it's not part of `require` but only `require-dev` to make it available to tests.